AdminWebsite/AdminWebsite/ClientApp/src/app/services/video-hearings.service.ts

```diff
@@ -432,6 +432,12 @@ export class VideoHearingsService {
         return this.modelHearing.participants.every(x => x.username.toLowerCase() !== username.toLowerCase());
     }
 
+    canAddJudge(username: string): boolean {
+        return !this.modelHearing.participants.some(
+            x => x.username?.toLowerCase() === username.toLowerCase() && this.judiciaryRoles.includes(x.hearing_role_name)
+        );
+    }
+
```

Comment:
As service is not updated while using assign-judge component this needs to check other participants. Otherwise changing judge and changing back to the "existing" judge in the service will throw an error